monitor/index.ts

```diff
@@ -2,21 +2,40 @@ import { SDK } from "codechain-sdk";
 import * as chainErrors from "./Errors";
 import { getConfig, sendNotice } from "./util";
 
-async function main() {
-  const rpcUrl = getConfig<string>("rpc_url");
-  const networkId = getConfig<string>("network_id");
-  const sdk = new SDK({ server: rpcUrl, networkId });
-
+const checkDeath = (() => {
   let prevBestBlockNumber = 0;
-
-  while (true) {
-    await new Promise(resolve => setTimeout(resolve, 60 * 60 * 1000));
+  return async (sdk: SDK) => {
     const currentBestBlockNumber = await sdk.rpc.chain.getBestBlockNumber();
     if (prevBestBlockNumber === currentBestBlockNumber) {
-      sendNotice(chainErrors.codechainDeath);
+      sendNotice(new chainErrors.CodeChainDeath());
     }
     prevBestBlockNumber = currentBestBlockNumber;
+  };
+})();
+
+async function checkView(sdk: SDK) {
+  const viewAlertLevel = 3;
```

Comment:
Would you make it configurable?

Reply:
Now viewAlertLevel is loaded from configuration